tests/functional.mk

```diff
@@ -79,7 +79,7 @@ $(call forward-vars,$(FTST_FUNCTIONAL_TESTS_STAMP), \
 $(FTST_FUNCTIONAL_TESTS_STAMP): $(FTST_IMAGE) $(FTST_EMPTY_IMAGE) $(ACTOOL_STAMP) $(RKT_STAMP) $(FTST_ACE_MAIN_IMAGE) $(FTST_ACE_SIDEKICK_IMAGE) $(FTST_SS1_STAMP) | $(FTST_TEST_TMP) $(FTST_RKT_PATH) $(FTST_STAGE1_ALL_FLAVOR_SYMLINKS)
 	$(VQ) \
 	$(call vb,vt,GO TEST,$(REPO_PATH)/tests) \
-	sudo RKT_STAGE1_DEFAULT_FLAVOR="$(RKT_STAGE1_DEFAULT_FLAVOR)" RKT="$(FTST_RKT_PATH)" ACTOOL="$(ACTOOL)" RKT_INSPECT_IMAGE="$(FTST_IMAGE)" RKT_EMPTY_IMAGE="$(FTST_EMPTY_IMAGE)" RKT_ACE_MAIN_IMAGE=$(FTST_ACE_MAIN_IMAGE) RKT_ACE_SIDEKICK_IMAGE=$(FTST_ACE_SIDEKICK_IMAGE) FUNCTIONAL_TMP="$(FTST_TEST_TMP)" INSPECT_BINARY="$(FTST_INSPECT_BINARY)" STUB_STAGE1="$(FTST_SS1_IMAGE)" $(GO_ENV) "$(ABS_GO)" test -tags $(RKT_STAGE1_DEFAULT_FLAVOR) -timeout 45m -v $(GO_TEST_FUNC_ARGS) $(REPO_PATH)/tests
+	sudo RKT_STAGE1_DEFAULT_FLAVOR="$(RKT_STAGE1_DEFAULT_FLAVOR)" RKT="$(FTST_RKT_PATH)" ACTOOL="$(ACTOOL)" RKT_INSPECT_IMAGE="$(FTST_IMAGE)" RKT_EMPTY_IMAGE="$(FTST_EMPTY_IMAGE)" RKT_ACE_MAIN_IMAGE=$(FTST_ACE_MAIN_IMAGE) RKT_ACE_SIDEKICK_IMAGE=$(FTST_ACE_SIDEKICK_IMAGE) FUNCTIONAL_TMP="$(FTST_TEST_TMP)" INSPECT_BINARY="$(FTST_INSPECT_BINARY)" STUB_STAGE1="$(FTST_SS1_IMAGE)" $(GO_ENV) "$(ABS_GO)" test -tags $(RKT_STAGE1_DEFAULT_FLAVOR) -timeout 60m -v $(GO_TEST_FUNC_ARGS) $(REPO_PATH)/tests
```

Comment:
As we keep bumping this and the line is non trivial, can we have this value in its own variable?